src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -1656,15 +1692,18 @@ static void Main()
                 expectedOutput: "value: 'value: '0', fromEnd: 'False'', fromEnd: 'value: '0', fromEnd: 'True''")
                 .VerifyIL("Program.Create", @"
 {
-  // Code size       20 (0x14)
-  .maxstack  1
+  // Code size       27 (0x1b)
+  .maxstack  3
   .locals init (System.Index V_0)
   IL_0000:  ldloca.s   V_0
   IL_0002:  initobj    ""System.Index""
   IL_0008:  ldloc.0
-  IL_0009:  call       ""System.Range System.Range.FromStart(System.Index)""
```

Comment:
>""System.Range System.Range.FromStart(System.Index)"" [](start = 23, length = 53)

Is there a design/speclet document that should be updated to reflect the compiler/API design?
It may be good to send an email to LDM (and relevant BCL folks) to let them know of this change and rationale.